Update statements.mdx with warning over iterating members of a struct (#246)
tact-lang/tact-docs: pages/book/statements.mdx

  }
  ```

> In a similar way, trying to iterate over a member of a struct won't work:

Reviewer: ```suggestion
  Trying to iterate over a member of a [Struct](/book/structs-and-messages#structs) also won't work:
```

>   Trying to iterate over a member of a [Struct](/book/structs-and-messages#structs) also won't work:

Reviewer 2: ```suggestion
  Trying to iterate over a member of a [Struct](/book/structs-and-messages#structs) also won't work, because struct field access is a compound expression and not an identifier:
```
